packages/react/src/utils/has-rsbuild-plugin.ts

```diff
@@ -0,0 +1,10 @@
+import { readNxJson, Tree } from '@nx/devkit';
+
+export function hasRsbuildPlugin(tree: Tree) {
+  const nxJson = readNxJson(tree);
+  return !!nxJson.plugins?.some((p) =>
+    typeof p === 'string'
+      ? p === '@nx/rsbuild/plugin'
```

Comment:
Rename to `@nx/rsbuild` if we change the inference plugin entry. If not, it should be a follow-up PR.

Reply:
Will be in a follow up